static/app/views/settings/organizationMembers/organizationMembersList.tsx

```diff
@@ -311,7 +311,7 @@ function OrganizationMembersList() {
           refetchInviteRequests();
           refetchMembers();
         }}
-        allowedRoles={currentMember ? currentMember.roles : ORG_ROLES}
+        allowedRoles={currentMember?.orgRoleList ?? currentMember?.roles ?? ORG_ROLES}
```

Comment:
Changes in this file should reflect the deprecation of `currentMember.roles` in favor of `currentMember.orgRoleList`